css/components/teaser.css

```diff
@@ -55,7 +55,7 @@
 .news-article .field--name-localgov-news-related > .field__item {
   margin-bottom: var(--vertical-rhythm-spacing);
   padding-bottom: var(--vertical-rhythm-spacing);
-  border-bottom: var(--border);
+  border-bottom: var(--service-cta-border);
```

Comment:
If we are going to set the border on teasers to something specific, let's called it `--border-teaser` or `--teaser-border`, and let that evaluate to `--border` in the `variables.css` file